src/main/java/eu/dissco/orchestration/backend/repository/SourceSystemRepository.java

```diff
@@ -14,14 +18,12 @@ public class SourceSystemRepository {
   private final DSLContext context;
 
   public int createSourceSystem(SourceSystemRecord sourceSystemRecord) {
-    return context.insertInto(NEW_SOURCE_SYSTEM)
-        .set(NEW_SOURCE_SYSTEM.ID, sourceSystemRecord.id())
+    return context.insertInto(NEW_SOURCE_SYSTEM).set(NEW_SOURCE_SYSTEM.ID, sourceSystemRecord.id())
```

Comment:
I don't think this has been changed.
Could you revert the change here and keep the indentation as it was.
For readability is it nicer if all the `.set` are on a newline